tools/webhook_test.py

```diff
@@ -43,7 +38,7 @@ def set_init(webhook_type):
                 "player_level": 30,
                 "latitude": 33.980823,
                 "longitude": -81.052988,
-                "encounter_id": 0,
+                "encounter_id": randint(1, 99999999),
```

Comment:
Why don't we just make this time based and remove the chance to repeats entirely?

Reply:
I agree with this, if you truly want to make it fully random and unique, generate a hash off of the current epoch